src/components/Course/stylesheet.scss

```diff
@@ -65,4 +66,54 @@
     display: flex;
     flex-direction: column;
   }
+  .course-info {
+    text-align: center;
+    background-color: rgb(50, 55, 58);
+    border-radius: 3px;
+    border-color: rgb(50, 55, 58);
+
+    .labelAverage {
+      font-weight: bold;
+      display: inline-block;
+      font-size: 20px;
+      height: 35px;
+      line-height: 28px;
+      color: white;
+    }
+
+    .gpa {
+      width: 50px;
+      height: 30px;
+      margin-left: 8px;
+      line-height: 28px;
+      text-align: center;
+      color: white;
+      font-weight: 500;
+      border: 2px solid #202020;
+      display: inline-block;
+      font-size: 18px;
+    }
+  }
+
+  .labelAverage {
+    font-weight: 500;
+    font-style: italic;
+    display: inline-block;
+    font-size: 13px;
```

Comment:
For `font-size`, use unit `em` instead of `px`. It is not an absolute standard, but to be consistent within this project.